explorer-ui/src/components/ProfileView/Preferences/Preferences.jsx

```diff
@@ -101,22 +129,22 @@ function handleOccupationsSelected(event, occupation) {
           }
           </div>
         </div>
-        <div className="tourist-attractions">
-          <h3>Select the following tourist attractions that interest you</h3>
-          <Activities country={"Moscow"}  handleAttractionsSelected={handleAttractionsSelected}/>
-        </div>
         <div className="curr-location">
-            <form className="set-location" onSubmit={(event) => handleLocationSet(event, country, province, city)}>
+            <form className="set-location" onClick={(event) => handleLocationSet(event, country, province, city)}>
           <h3>Where are you currently located</h3>
           <span>Country</span>
           <input ref={country}className="country"></input>
           <span>State/Province</span>
           <input ref={province}className="state"></input>
           <span>City/Town</span>
           <input ref={city}className="city"></input>
-          <button type="submit">Submit</button>
+          <button type="click">Submit</button>
           </form>
         </div> 
+        <div className="tourist-attractions">
+          <h3>Select the following tourist attractions that interest you</h3>
+          <Activities country={"Moscow"}  handleAttractionsSelected={handleAttractionsSelected}/>
```

Comment:
nit - remove brackets "Moscow"

Reply:
code is now obsolete since i'm passing in props to this field